app/View/Pages/help.ctp

```diff
@@ -107,10 +67,16 @@ $this->set('title_for_layout', $this->Pages->formatTitle(__('Help')));
                 __(
                     'If you have no idea what to do now that you are registered, '.
                     'you can introduce yourself on the <a href="{}">Wall</a>, or '.

```

Comment:
You should change `<a href="{}">Wall</a>` to `<a href="{wall}">Wall</a>` in order to refer to it below.